dags/vz_afd_ems_import.py

```diff
@@ -0,0 +1,84 @@
+# stuff to make the airflow, 1Password integration work
+import os
+import pendulum
+import docker
+from airflow.decorators import dag, task
+from onepasswordconnectsdk.client import Client, new_client
```

Comment:
@frankhereford looks like you can remove the imports from `onepasswordconnectsdk`.

Reply:
Great call, thank you John!